packages/api/src/utils/query-utils.ts

```diff
@@ -7,7 +7,7 @@ export const getQueryKeys = (name: string, props?: Record<string, unknown>) => {
     if (Object.keys(props).length === 0) return [name];
 
     const ordered_props = Object.keys(props)
-        .sort()
+        .sort((a, b) => a.localeCompare(b))
         .reduce((obj, key) => {
```

Comment:
Oh wait I forgott, we should add an `index.ts` file into `utils` to minimize the circular dependencies issues:
```suggestion
 export * from './file-utils'
 export * from './query-utils'
```